pkg/config/reader.go

```diff
@@ -35,5 +37,23 @@ func (jcr *JobConfigReader) GetJobConfig() (*Config, error) {
 		log.Printf("The config was: %s", string(resource))
 		return nil, fmt.Errorf("error parsing job configuration: %w", err)
 	}
+
+	// Validate / Correct the Job cleanup TTL to 60s to avoid a job cleanup before the
```

Comment:
As a matter of responsibility, I would have put this snippet in the k8sutils package and verified that we enforce the minimum TTL when creating the job on k8s (the jobconfig reader has the responsibility of retrieving and parsing the config yaml, while the minimum ttl is needed by the k8s polling mechanism in k8sutils).

If the implementation for checking the job status changes we now have 2 packages impacted instead of only one.